packages/contracts/package.json

```diff
@@ -33,6 +33,7 @@
         "@unirep/utils": "2.0.0-beta-1",
         "@zk-kit/incremental-merkle-tree.sol": "^1.4.0",
         "ethers": "^5.6.8",
+        "global-factory": "^0.0.0",
```

Comment:
there is a more latest version of `global-factory`?

Reply:
Yes there is a later patch version, but the only changes are `README` and `package.json`, adding keywords and repo url